services/tap/values-usdfdev.yaml

```diff
@@ -0,0 +1,12 @@
+qserv:
+  host: "172.24.4.101:4040"
+  mock:
+    enabled: false
+
+config:
+  gcsBucket: "rubin-qserv"
+  gcsBucketUrl: "https://s3dfrgw.slac.stanford.edu"
+  gcsBucketType: "S3"
+  jvmMaxHeapSize: "31G"
+  http_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
```

Comment:
Nothing uses this setting so far as I can see. If you need to set an HTTP proxy, I think it needs to be plumbed through to the resources.